x/stake/client/cli/flags.go

```diff
@@ -45,9 +45,9 @@ func init() {
 	fsDescriptionCreate.String(FlagWebsite, "", "optional website")
 	fsDescriptionCreate.String(FlagDetails, "", "optional details")
 	fsDescriptionEdit.String(FlagMoniker, types.DoNotModifyDesc, "validator name")
-	fsDescriptionEdit.String(FlagIdentity, types.DoNotModifyDesc, "optional identity signature (ex. UPort or Keybase)")
-	fsDescriptionEdit.String(FlagWebsite, types.DoNotModifyDesc, "optional website")
-	fsDescriptionEdit.String(FlagDetails, types.DoNotModifyDesc, "optional details")
+	fsDescriptionEdit.String(FlagIdentity, types.DoNotModifyDesc, "identity signature (ex. UPort or Keybase)")
```

Comment:
Took a look at #1957 -- what is the motivation for requiring all fields? Wouldn't that make it burdensome for the user when updating their validator? e.g. I want to just update my website to use HTTPS, but I have to also port over my name, details (which could be long?).

If that's what we want, then it seems this PR covers it 👍

/cc @jackzampolin

Reply:
See issue.